app/modules/intelligence/agents_copy/chat_agents/adaptive_agent.py

```
@@ -0,0 +1,115 @@
+from enum import Enum
+
+from app.modules.intelligence.agents_copy.chat_agent import ChatAgentResponse
+from ..chat_agent import ChatAgent
+from .simple_llm_agent import SimpleLLMAgent
+from app.modules.intelligence.provider.provider_service import (
+    ProviderService,
+    AgentType,
+)
+from app.modules.intelligence.prompts.classification_prompts import (
+    ClassificationPrompts,
+    AgentType,
```

Ruff (0.8.2)

Redefinition of unused `AgentType` from line 10

(F811)